integration/mesos/src/main/java/alluxio/mesos/AlluxioScheduler.java

```diff
@@ -265,11 +265,7 @@ public void resourceOffers(SchedulerDriver driver, List<Protos.Offer> offers) {
    *         current configuration
    */
   private String createAlluxioSiteProperties() {
-    StringBuilder siteProperties = new StringBuilder();
-    for (Entry<String, String> entry : Configuration.toMap().entrySet()) {
-      siteProperties.append(String.format("%s=%s%n", entry.getKey(), entry.getValue()));
-    }
-    return siteProperties.toString();
+    return Configuration.getSitePropertiesFile();
```

Comment:
we can remove this method and replace the call by calling `Configuration.getSiteProperties` directly